templates/template-for-tasks.rst

```diff
@@ -0,0 +1,171 @@
+:orphan: true
```

Comment:
Right, i had inserted it temporarily to suppress the warnings that the file isn't included in any toctree when we build in this dir, until we move it to the right place but i removed the line, and we'll just be aware of this being a temporary issue.